app/javascript/components/assessment/steps/SymptomsAssessment.js

```diff
@@ -59,7 +59,7 @@ class SymptomsAssessment extends React.Component {
     const keysToIgnore = ['who_reported'];
     let allFieldsEmpty = true;
     _.map(object, (value, key) => {
-      if (object[String(key)] !== null && !keysToIgnore.includes(key)) {
+      if (object[String(key)] && !keysToIgnore.includes(key)) {
```

Comment:
Is the idea for this change to also skip undefined (in addition to null)? Seems like a good idea to me.

Reply:
Correct, in this case we only care about values that are true

Reply:
At first I thought this would be problematic but the values at this point at "Yes"/"No" rather than true/false so it works out. Leaving a comment in case others have the same concern at first.

Reply:
Actually though just to be safe because I think technically float/integer fields with value of 0 are not truthy and would not pass this, can you change this to `object[String(key)] !== null && object[String(key)] !== undefined` @mdube7 ?

Reply:
Made the suggested change